python/ray/tune/impl/tuner_internal.py

```diff
@@ -99,20 +99,21 @@ def __init__(
                     "Tuner(..., run_config=RunConfig(...))"
                 )
 
+        self.trainable = trainable
+        self.param_space = param_space or {}
```

Comment:
Should this be 
```suggestion
        self._param_space = param_space or {}
```
? (because we do have the property below)